src/main/java/core/basesyntax/utility/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.utility;
+
+public class RegistrationException extends RuntimeException {
```

Comment:
The class `RegistrationException` is currently extending `RuntimeException` which is an unchecked exception. If your application should catch exceptions, consider making `RegistrationException` extend from `Exception` (a checked exception) instead. However, if these exceptions are something that will cause your application to stop, then it is alright to use `RuntimeException`.